src/main/java/nl/tudelft/contextproject/gui/PresetController.java

```diff
@@ -69,14 +69,23 @@ private void initialize() {
         applySettings();
         setFactories();
         setActions();
+        
+        tableView.getSortOrder().clear();
+        tableView.getSortOrder().add(presetColumn);
```

Comment:
I didn't create a method for those lines because it are just two lines. But, readability is a good argument so I will creat the method.